<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountLoginDialog</class>
 <widget class="QDialog" name="AccountLoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>212</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add Account</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="typeBox"/>
   </item>
   <item>
    <widget class="QLabel" name="errorLbl">
     <property name="text">
      <string>Error label placeholder.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="infoLbl">
     <property name="text">
      <string>Message label placeholder.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="userTextBox">
     <property name="placeholderText">
      <string>Email / Username</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passTextBox">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ProgressWidget" name="progressWidget" native="true"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProgressWidget</class>
   <extends>QWidget</extends>
   <header>widgets/ProgressWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
